doc/source/changes/version_0_28.rst.inc

```diff
@@ -212,6 +212,10 @@ Miscellaneous improvements
      a1   1   1
 
 
+* renamed argument `nb_index` of `read_csv`, `read_excel`, `read_sas`, `from_lists` and `from_string` functions
+  as `nb_axes` (closes :issue:`548`).
```

Comment:
this description does not make it clear enough that you have to change the argument name **and change its value**.